clippy_dev/src/fmt.rs

```diff
@@ -64,7 +65,11 @@ pub fn run(check: bool, verbose: bool) {
                 continue;
             }
 
-            success &= rustfmt(context, path)?;
+            paths.push(entry.into_path());
+        }
+
+        for chunk in paths.chunks(250) {
+            success &= rustfmt(context, chunk)?;
```

Comment:
I think we should avoid collecting *all* the paths into the Vec at once. You could use `Itertools::chunks`.

Reply:
👍 changed